frappe/database/database.py

```diff
@@ -549,7 +549,7 @@ def get_values_from_single(
 				return r and [[i[1] for i in r]] or []
 
 
-	def get_singles_dict(self, doctype, debug = False):
+	def get_singles_dict(self, doctype, debug=False, *, for_update=False):
```

Comment:
I have kept the `debug` parameter as-is for compatibility but it isn't being used.